aten/src/ATen/native/transformers/cuda/flash_attn/flash_api.h

```diff
@@ -1,10 +1,11 @@
 #pragma once
 #include <cstddef>
 
+#include <namespace_config.h>
```

Comment:
Do you need a separate config file here versus some other existing standard PyTorch config header?

Reply:
So this config header is this one: https://github.com/Dao-AILab/flash-attention/pull/1419/files#diff-1271a158569ddafcee0391c481c061a54b79bd61dba1c32c0514a5182b4f4e75 which probably has more extras then needed